src/pages/junghoon/Login/Login.js

```diff
@@ -1,13 +1,108 @@
-import React, { Component } from 'react'
+import React, { Component } from "react";
+import "./Login.scss";
+import "../../../styles/common.scss";
 
 class Login extends Component {
+  constructor(e) {
+    super(e);
+
+    this.state = {
+      userId: "",
+      userPw: "",
+    };
+  }
+
+  // goToMain = () => {
+  //   this.props.history.push("/mainPark");
+  // };
+
+  handleIdInput = (e) => {
+    this.setState({
+      userId: e.target.value,
+    });
+  };
+
+  handlePwInput = (e) => {
+    this.setState({
+      userPw: e.target.value,
+    });
+  };
```

Comment:
```suggestion
  handleInputs = e => {
    const { name, value } = e.target;
    
    this.setState({
      [name]: value,
    });
  };
```
같은 기능의 함수를 하나로 합쳐주세요!